apps/web/src/lib/auth/index.ts

```diff
@@ -0,0 +1,36 @@
+import { sessions, users } from '@latitude-data/core'
+import { DrizzlePostgreSQLAdapter } from '@lucia-auth/adapter-drizzle'
+import db from '$/db/database'
```

Comment:
hmmm I've removed this code in my PR, since we shouldn't export db to consumers imo